packages/office-ui-fabric-react/src/components/TextField/TextField.scss

```diff
@@ -71,7 +71,7 @@
   font-size: $ms-font-size-m;
   color: $ms-color-neutralPrimary;
   height: 32px;
-  @include padding(0, 12px, 0, 12px);
+  @include padding(0, 12px, 1px, 12px);
```

Comment:
1px bottom padding seems to be our consistent text field v alignment. I.E. line up dropdown, text field, search and picker and this 1px makes all 4 align.